usage:
```
vsg provides CMake targets:

    # this is heuristically generated, and may not be correct
    find_package(vsg CONFIG REQUIRED)
    target_link_libraries(main PRIVATE vsg::vsg)
```

When testing usage, the following error occurs:
```
1> [CMake] CMake Error at E:/vsg/installed/x64-windows/share/vsg/vsgTargets.cmake:60 (set_target_properties):
1> [CMake]   The link interface of target "vsg::vsg" contains:
1> [CMake] 
1> [CMake]     glslang::glslang
1> [CMake] 
1> [CMake]   but the target was not found.  Possible reasons include:
1> [CMake] 
1> [CMake]     * There is a typo in the target name.
1> [CMake]     * A find_package call is missing for an IMPORTED target.
1> [CMake]     * An ALIAS target is missing.
1> [CMake] 
1> [CMake] Call Stack (most recent call first):
1> [CMake]   E:/vsg/installed/x64-windows/share/vsg/vsgConfig.cmake:7 (include)
1> [CMake]   E:/vsg/scripts/buildsystems/vcpkg.cmake:853 (_find_package)
1> [CMake]   CMakeFindUsage/CMakeLists.txt:17 (find_package)
1> [CMake] -- Generating done (0.0s)
1> [CMake] CMake Generate step failed.  Build files cannot be regenerated correctly.
```

<details>
<summary><b>CMakeFindUsage.cpp</b></summary>
<pre>
#include &#60;iostream&#62;
#include "vsg/all.h"

using namespace std;

int main()
{
        cout << "Hello CMake." << endl;
        return 0;
}
</pre></details>


<details>
<summary><b>CMakeLists.txt</b></summary>
<pre>
cmake_minimum_required (VERSION 3.8)

set(CMAKE_TOOLCHAIN_FILE "E:/vsg/scripts/buildsystems/vcpkg.cmake")

project ("CMakeFindUsage")

add_executable (main "CMakeFindUsage.cpp")

find_package(vsg CONFIG REQUIRED)
target_link_libraries(main PRIVATE vsg::vsg)
</pre></details>

--- a/ports/vsg/devendor-glslang.patch
+++ b/ports/vsg/devendor-glslang.patch
@@ -16,10 +16,10 @@ index 6e7ec437..db4d30f1 100644
  
          if (Git_FOUND)
 diff --git a/src/vsg/CMakeLists.txt b/src/vsg/CMakeLists.txt
-index 3d9625e4..8fe89f89 100644
+index 3d9625e4..96285e99 100644
 --- a/src/vsg/CMakeLists.txt
 +++ b/src/vsg/CMakeLists.txt
-@@ -227,7 +227,7 @@ set(SOURCES
+@@ -227,14 +227,24 @@ set(SOURCES
      utils/LoadPagedLOD.cpp
  )
  
@@ -28,10 +28,12 @@ index 3d9625e4..8fe89f89 100644
  
      # include glslang source code directly into the VulkanScenegraph library build.
      include(../glslang/build_vars.cmake)
-@@ -235,6 +235,14 @@ endif()
+ endif()
  
  # set up library dependencies
- set(LIBRARIES PUBLIC
+-set(LIBRARIES PUBLIC
++set(LIBRARIES
++	PRIVATE
 +        glslang::glslang
 +        glslang::OSDependent
 +        glslang::MachineIndependent
@@ -40,10 +42,11 @@ index 3d9625e4..8fe89f89 100644
 +        glslang::OGLCompiler
 +        glslang::SPVRemapper
 +        glslang::SPIRV
++	PUBLIC
          Vulkan::Vulkan
          Threads::Threads
  )
-@@ -365,9 +373,6 @@ target_include_directories(vsg
+@@ -365,9 +375,6 @@ target_include_directories(vsg
      PUBLIC
          $<BUILD_INTERFACE:${VSG_SOURCE_DIR}/include>
          $<BUILD_INTERFACE:${VSG_BINARY_DIR}/include>

--- a/ports/vsg/devendor-glslang.patch
+++ b/ports/vsg/devendor-glslang.patch
@@ -69,3 +69,15 @@ index 71a7f09f..803f26a1 100644
  #    include <glslang/Public/ResourceLimits.h>
  #    include <glslang/Public/ShaderLang.h>
  #endif
+diff --git a/src/vsg/vsgConfig.cmake.in b/src/vsg/vsgConfig.cmake.in
+index 7ea0de02..4f7e77bd 100644
+--- a/src/vsg/vsgConfig.cmake.in
++++ b/src/vsg/vsgConfig.cmake.in
+@@ -2,6 +2,7 @@ include(CMakeFindDependencyMacro)
+ 
+ find_package(Vulkan @Vulkan_MIN_VERSION@ REQUIRED)
+ find_dependency(Threads)
++find_dependency(glslang)
+ 
+ @FIND_DEPENDENCY_WINDOWING@
+ include("${CMAKE_CURRENT_LIST_DIR}/vsgTargets.cmake")

--- a/versions/v-/vsg.json
+++ b/versions/v-/vsg.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f20a9579b1ddebc5321d5177f3604ed74965abd3",
+      "git-tree": "083510fe30c973a53fe65f4565d7c4159fdff36a",
       "version": "1.0.6",
       "port-version": 0
     },

--- a/versions/v-/vsg.json
+++ b/versions/v-/vsg.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "a28fafb639c82c5073955269eb3f931e41d3b17a",
+      "git-tree": "f20a9579b1ddebc5321d5177f3604ed74965abd3",
       "version": "1.0.6",
       "port-version": 0
     },

